cmd/scollector/collectors/snmp_fortinet.go

```diff
@@ -0,0 +1,117 @@
+package collectors
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+const (
```

Comment:
@captncraig  That would be change form all the other snmp collectors. We can do that, but would rather it be a commit changing all of them.